pkg/cli/log.go

```diff
@@ -17,16 +18,24 @@ func NewLogs() *cobra.Command {
 }
 
 type Logs struct {
-	Follow bool `short:"f" usage:"Follow log output"`
+	Follow    bool  `short:"f" usage:"Follow log output"`
+	TailLines int64 `short:"n" usage:"Number of lines in log output"`
```

Comment:
Can you change the field name to be `Tail` to be consistent with `docker` and `kubectl` CLIs.

Reply:
Should be changed now, also added --since flag for time based filtering, I'm unsure why the make validate-ci is failing however any thoughts @ibuildthecloud?

Reply:
Figured out why it was failing forgot to add the Since schema to the openapi generated file